### DISCUSSION TITLE
Home Assistant OS 11.1

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2866] [Announcements]
## Home Assistant Operating System

* Prepare 11.1.rc1 (#2859) @agners
* bump generic_raw_uart to 1.30 (#2847) @jens-maus
* Fix IPv6 routing failover for Thread (#2434) (#2845) @agners
* Bump buildroot to update BlueZ to v5.70 (#2835) @sairon
* Bump buildroot to 2023.02.6 (#2831) @agners
* Fix deployment on dev branch (#2834) @agners

## Build System

* Add action for bumping OS version for RPi imager (#2861) @sairon
* Do not use working-directory in test job (#2858) @sairon
* Use alternative environment for release build bump (#2855) @sairon
* Fix extraction of OVA image artifact in test step (#2842) @sairon
* Fix path to test results in processing steps (#2837) @sairon
* Fix test action by checking out source first (#2836) @sairon
* Create foundation for Labgrid-based OS tests (#2812) @sairon
* Add Home Assistant Green to GitHub infrastructure (#2828) @agners
* Use Docker 24 during build time (#2811) @agners

## Open Virtual Appliance

* Linux: Update kernel 6.1.59 (#2849) @sairon
* Bump qemu-guest-agent to 8.0.5 (#2846) @jens-maus

## Generic x86-64

* Linux: Update kernel 6.1.59 (#2849) @sairon

## Hardkernel ODROID

* ODROID-M1: Fix environment read from SD card (#2822) (#2856) @agners
* Linux: Update kernel 6.1.59 (#2849) @sairon

## ASUS Tinker

* Linux: Update kernel 6.1.59 (#2849) @sairon

## Khadas VIM Series

* Linux: Update kernel 6.1.59 (#2849) @sairon

## Generic aarch64

* Linux: Update kernel 6.1.59 (#2849) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.1'>Home Assistant OS 11.1</a>.</em>

**Comment by Welsyntoffie** [comment]
Strange thing happening. Every time I upgrade from version 10.5 about, my CPU keeps increasing usage. 
Was at 41% last night. Upgraded this morning to 11.1 and now I sit at 71% constant.
I also see that tailscale is no longer playing along as its hovering at 24%.

**Comment by zhujy09** [comment]
I got a Home Assistant GREEN recently. For some reason I would like to reset/reinstall the OS.
Due to the poor network condition of connecting to Github in CN Mainland, it seems not work to reset the GREEN via this [official guide](https://green.home-assistant.io/guides/reset/).
Is there a way to install the HAOS fully offline via the OS image here "[haos_generic-x86-64-11.1.img.xz](https://github.com/home-assistant/operating-system/releases/download/11.1/haos_generic-x86-64-11.1.img.xz)" ? How to achieve that?

**Reply by Welsyntoffie** [reply]
Try home assistant reddit, I dont think anyone uses these discussions
